material_maker/nodes/generic/generic.gd

```diff
@@ -494,21 +494,27 @@ func save_generator() -> void:
 	while files is GDScriptFunctionState:
 		files = yield(files, "completed")
 	if files.size() > 0:
-		do_save_generator(files[0])
-
-func do_save_generator(file_name : String) -> void:
+		do_save_generator(files[0], generator)
+
+static func extract_verbose_values(path : String, data : Dictionary) -> Array:
```

Comment:
This can be moved to `loader.gd` now that `save_gen()` is there, although I think it's better to remove it along with `string_to_dict_tree_old()` unless there's a need to keep it?